src/plugins/timelion/server/saved_objects/timelion_sheet.ts

```diff
@@ -12,6 +12,20 @@ export const timelionSheetSavedObjectType: SavedObjectsType = {
   name: 'timelion-sheet',
   hidden: false,
   namespaceType: 'single',
+  management: {
+    icon: 'visTimelion',
+    defaultSearchField: 'title',
+    importableAndExportable: true,
+    getTitle(obj) {
+      return obj.attributes.title;
+    },
+    getInAppUrl(obj) {
+      return {
+        path: `/app/timelion#/${encodeURIComponent(obj.id)}`,
+        uiCapabilitiesPath: 'visualize.show',
```

Comment:
just have a question about **uiCapabilitiesPath**. For **timelion** I see capabilities only for **save**. Decided to use **visualize.show** for that case

Reply:
@alexwizp tbh I think that `visualize:show` is not the correct capabilities here. Timelion is a different application. I can see that during the setup we register timelion capabilities
`core.capabilities.registerProvider(() => ({
      timelion: {
        save: true,
      },
    }));`

Can we add a new capability for show here? Wdyt?

Reply:
agree, done